cdap-ui/app/cdap/components/Overview/OverviewHeader/index.js

```diff
@@ -41,7 +41,7 @@ export default class OverviewHeader extends Component {
   }
   render() {
     return (
-      <div className="overview-header">
+      <div className={classnames("overview-header", {'message-placeholder': typeof this.props.onClose !== 'function'})}>
```

Comment:
In what case would `message-placeholder` show up? I'm trying to understand the purpose of this addition, and also the `this.props.onClose?` check a few lines below.